openmeter/billing/invoiceline.go

```diff
@@ -151,7 +151,8 @@ type LineBase struct {
 	Status                 InvoiceLineStatus `json:"status"`
 	ChildUniqueReferenceID *string           `json:"childUniqueReferenceID,omitempty"`
 
-	TaxConfig *TaxConfig `json:"taxOverrides,omitempty"`
+	TaxConfig         *TaxConfig               `json:"taxOverrides,omitempty"`
+	RateCardDiscounts productcatalog.Discounts `json:"rateCardDiscounts,omitempty"`
```

Comment:
We would collide with the Line's Discounts. For now this is fine, but most probably the whole RateCard should move to the type specific entity.